mobile/src/main/java/ch/epfl/sdp/mobile/application/social/SocialFacade.kt

```diff
@@ -40,20 +41,19 @@ class SocialFacade(private val auth: Auth, private val store: Store) {
   /**
    * Returns a [Flow] of the [Profile] by the given name
    *
-   * @param name text search criteria.
+   * @param uid search criteria.
    * @param user the [AuthenticationUser] that is performing the get.
    */
   fun get(
```

Comment:
Already in another PR's comments I think, but this should maybe be called `profile(...)`, since `get` is very generic, especially in a `SocialFacade`.